### PR TITLE
Checks if family has location

### PR DESCRIPTION
**Pull request by paulinembabu**

### CODE REVIEW

File: opensrp-chw/src/main/java/org/smartregister/chw/activity/AllClientsMemberProfileActivity.java [hunk 1]
```diff
@@ -196,6 +196,11 @@ protected void startMalariaFollowUpVisit() {
         MalariaFollowUpVisitActivity.startMalariaFollowUpActivity(this, baseEntityId);
     }
 
+    @Override
+    protected void setIndependentClient(boolean b) {
```
Comment (keymane) [inline opensrp-chw/src/main/java/org/smartregister/chw/activity/AllClientsMemberProfileActivity.java:200]: Codacy Issue found: [Document empty method body](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5517546)

**Comment by keymane** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5517546&bid=18078876)

File: opensrp-chw/src/main/java/org/smartregister/chw/activity/FamilyOtherMemberProfileActivity.java [hunk 2]
```diff
@@ -184,4 +184,8 @@ protected BaseFamilyOtherMemberProfileFragment getFamilyOtherMemberProfileFragme
     protected void startMalariaFollowUpVisit() {
         MalariaFollowUpVisitActivity.startMalariaFollowUpActivity(this, baseEntityId);
     }
+
+    @Override
+    protected void setIndependentClient(boolean b) {
```
Comment (keymane) [inline opensrp-chw/src/main/java/org/smartregister/chw/activity/FamilyOtherMemberProfileActivity.java:189]: Codacy Issue found: [Document empty method body](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5517546)

File: opensrp-chw/src/ba/java/org/smartregister/chw/activity/AncMemberProfileActivityFlv.java [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 public class AncMemberProfileActivityFlv extends DefaultAncMemberProfileActivityFlv {
     @Override
-    public Boolean flvSetFamilyLocation() {
+    public Boolean hasFamilyLocationRow() {
```
Comment (manutarus) [inline opensrp-chw/src/ba/java/org/smartregister/chw/activity/AncMemberProfileActivityFlv.java:5]: is it possible to check if the memberObject has gps at this point?

Comment (paulinembabu) [reply opensrp-chw/src/ba/java/org/smartregister/chw/activity/AncMemberProfileActivityFlv.java:5]: How?

**Pull request merged**